src/icons/fileIcons.ts

```diff
@@ -316,6 +315,9 @@ export const fileIcons: FileIcons = {
         'psql',
         'db',
         'db3',
+        'dblite',
```

Comment:
Should we use the "settings" icon here or rather the "database" icon?

Reply:
They're better suited as databases imo ("db" is in the name for two of them), so they're best left as-is under the "database" category

Reply:
Can you change it then?

Reply:
Ooh, hold up, I see what's going on. To clarify, they're already databases! The changelog formatting makes it look like they're under "settings" when compressed, so here's the expanded view:
![24-01-22 09-48-25 chrome](https://github.com/PKief/vscode-material-icon-theme/assets/17489292/58fa77c3-7e29-4633-aec8-b14e7ce4dacf)

Reply:
Oups, okay. Did't see it 🙈 Then it's okay!